server/plugins/openAiWhisperPlugin.js

```diff
@@ -39,6 +39,12 @@ function alignSubtitles(subtitles, format) {
         const subtitle = subtitles[i];
         result.push(...shiftSubtitles(subtitle, i * offset));
     }
+    
+    //if content has needed html style tags, keep them
+    for(const obj of result) {
```

Comment:
alignSubtitles could use some error handling.  You're having a lot of faith here that result is in the right form.

Reply:
added try-catch and obj field checking